packages/synapse-react-client/src/synapse-queries/file/FileUploadArgs.ts

```diff
@@ -0,0 +1,9 @@
+export type FileUploadArgs = {
```

Comment:
This type represents the shared arguments for `useDirectUploadToS3` in this PR, and another hook to use Synapse multipart upload coming in a later PR.